heidrun/lc_json.rb

```diff
@@ -0,0 +1,119 @@
+build_uri = lambda do |node|
+  uri = RDF::URI(node.value)
+  uri.scheme = 'http' unless uri.scheme
```

Comment:
No reason in particular. 

This is here because many of the IRIs are "//loc.gov/...". There's no reason not to force https for these cases, assuming we are confident it will be supported.